CriticalMass/MapInfoViewController.swift

```diff
@@ -117,6 +140,20 @@ class MapInfoViewController: UIViewController, IBConstructable {
             animator.startAnimation()
         }
     }
+
+    @objc private func didReceiveLocationsUpdate(notification: Notification) {
+        guard
+            let result = notification.object as? ApiResponseResult,
+            result.isError().0 != showServerError
+        else {
+            return
+        }
+        let errorMessage = (result.isError().1 as! NetworkError).errorDescription ?? L10n.Map.Layer.Info.errorMessage
```

Comment:
I received a `fatalError` when I tried to play around with the error view. I disconnected the device from internet connection and connected it again. It then crashed because `result.isError().1` is nil in that case. There is no error any more. So this should be better: 
```suggestion
        let errorMessage = (result.isError().1 as? NetworkError)?.errorDescription ?? L10n.Map.Layer.Info.errorMessage
```